lesson_3/ex1_solution.md

```diff
@@ -1,11 +1,14 @@
 
-- Image Size: 
-- Container Name: 
-- Container ID: 
-- Exposed port: 
-- Postgres version: 
-- Data Volume (aka PGDATA): 
-- default entrypoint: 
-- default command: 
-- baseline consumption : CPU ___% MEM ___MB  
+- Image Size: **377MB**
+- Container Name: **naughty_faraday**
+- Container ID: **4a5567f999e8ed59f8d38559d1e1f3a937181dcf89253645cd4292144663c561**
+- Exposed port: **{"5432/tcp":{}}**
+- Postgres version: **15.0-1.pgdg110+1**
+- Data Volume (aka PGDATA): **PGDATA=/var/lib/postgresql/data**
+- default entrypoint: **docker-entrypoint.sh**
+- default command: **"/bin/sh",
+                "-c",
+                "#(nop) ",
+                "CMD [\"postgres\"]"**
+- baseline consumption : CPU **0.03**% MEM **30.9**MB  
 - OS: Distro Version Codename (Ubuntu 22.04 jammy)
```

Comment:
necesitas entrar no contedor
docker exec -it naughty_faraday /bin/bash
more /etc/issue